website/util/rubeus.py

```diff
@@ -266,21 +267,14 @@ def collect_addon_js(node, visited=None, filename='files.js', config_entry='file
     :return list: List of JavaScript include paths
 
     """
-    # NOTE: must coerce to list so it is JSON-serializable
-    visited = visited or []
-    visited.append(node._id)
-    js = set()
-    for addon in node.get_addons():
+    js = []
+    for addon_config in settings.ADDONS_AVAILABLE_DICT.values():
```

Comment:
I'm not super-familiar with the addon system. Is is definitely equivalent to `node.get_addons()`?